src/OpenTelemetry.Exporter.Geneva/ExceptionStackExportMode.cs

```diff
@@ -14,9 +14,17 @@ public enum ExceptionStackExportMode
     Drop,
 
     /// <summary>
-    /// Exception stack traces are exported as strings.
+    /// Exception stack traces are exported as string, using a culture agnostic representation of the results of ToString() implementation of the Exception.
+    /// Though formatted in culture-agnostic way, this is more targetted towards human readability.
+    /// See <see href="https://learn.microsoft.com/dotnet/api/system.exception.stacktrace"/>.
     /// </summary>
     ExportAsString,
 
+    /// <summary>
+    /// Exception stack traces are exported as string, using the StackTrace property of the Exception.
```

Comment:
need to mention this does not do anything with InnerException or AggregateExcetion etc..where ToString() typically covers them.

Reply:
Also, need to confirm if this eliminates stacks for async/await machinery, which ToString() eliminates by default now.